connect/cli/plugins/project/extension_helpers.py

```diff
@@ -245,6 +383,22 @@ def _check_statuses(capability, stat, errors):
     ):
         if stat not in CAPABILITY_ALLOWED_STATUSES + REQUESTS_SCHEDULED_ACTION_STATUSES:
             errors.append(f'Status `{stat}` on capability `{capability}` is not allowed.')
+
+    elif capability in (
+        'listing_new_request_processing',
+        'listing_remove_request_processing',
+    ):
+        if stat not in LISTING_REQUEST_STATUSES:
+            errors.append(f'Status `{stat}` on capability `{capability}` is not allowed.')
+    elif capability == 'tier_account_update_request_processing':
+        if stat not in TIER_ACCOUNT_UPDATE_STATUSES:
+            errors.append(f'Status `{stat}` on capability `{capability}` is not allowed.')
+    elif capability == 'part_usage_file_request_processing':
+        if stat not in CHUNK_FILE_STATUSES:
+            errors.append(f'Status `{stat}` on capability `{capability}` is not allowed.')
+    elif capability == 'usage_file_request_processing':
+        if stat not in USAGE_FILE_STATUSES:
+            errors.append(f'Status `{stat}` on capability `{capability}` is not allowed.')
```

Comment:
Can you please simplify / extract?
https://sonarcloud.io/project/issues?id=connect-cli&open=AXzmfRmI9W6MU9yIVBwu&pullRequest=72&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL

Reply:
should be fixed, thanks